src/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+document.querySelector('#start-clock').addEventListener('click', () => {
+  document.querySelector('.watch__minute-hand').classList.add('start-minute');
+});
+
+document.querySelector('#start-clock').addEventListener('click', () => {
+  document.querySelector('.watch__second-hand').classList.add('start-second');
+});
```

Comment:
```suggestion
document.querySelector('#start-clock').addEventListener('click', () => {
  document.querySelector('.watch__minute-hand').classList.remove('pause');
  document.querySelector('.watch__second-hand').classList.remove('pause');
  
  document.querySelector('.watch__minute-hand').classList.add('start-minute');
  document.querySelector('.watch__second-hand').classList.add('start-second');
});

```

Reply:
And move references to the DOM elements to the variables. for example `const startButton = document.querySelector('#start-clock') `